server/api/googleLogin.js

```diff
@@ -18,7 +18,7 @@ async function googleLogin(req, res, next) {
   const username = req.body.googleIdToken;
```

Comment:
`username` -> `oauth_id` bit of a semantics thing, but looks good otherwise

Reply:
make sure to update it everywhere else in this file too if you update it. I would send back a `null` username in the response for now, and we probably don't need the `oauth_id` on the front end